include/picongpu/particles/bremsstrahlung/Bremsstrahlung.tpp

```diff
@@ -150,8 +151,13 @@ float3_X Bremsstrahlung<T_IonSpecies, T_ElectronSpecies, T_PhotonSpecies>::scatt
 }
 
 template<typename T_IonSpecies, typename T_ElectronSpecies, typename T_PhotonSpecies>
+template<typename T_Acc>
 DINLINE
-unsigned int Bremsstrahlung<T_IonSpecies, T_ElectronSpecies, T_PhotonSpecies>::numNewParticles(FrameType& sourceFrame, int localIdx)
+unsigned int Bremsstrahlung<T_IonSpecies, T_ElectronSpecies, T_PhotonSpecies>::numNewParticles(
```

Comment:
OPTIONAL:
according to the [coding guidelines](https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#17-template-class--struct-and-type-definitions) each template argument should be on a separate line

Reply:
Because I changed only a view lines within the code I tried to keep the style as near as possible to the old one. But in the case I changed the style to avoid endless long code lines.